### DISCUSSION TITLE
Patches and hotfixes

### DISCUSSION THREAD

**Discussion by charterchap** [CycloneDX/specification#81] [Q&A]
Is there a recommend way to list hotfixes. My specific example is Windows OS with their Quick Fix updates ([example](https://support.microsoft.com/en-us/topic/november-19-2020-kb4594440-os-builds-19041-631-and-19042-631-out-of-band-0b3ef07e-dbac-d445-7e90-80a717b101e0)).

I was thinking I could use pedigree:variants or pedigree:descendants but those are both arrays of components and the "type enum" options don't really apply. Is anyone doing this already on their SBOMs?

**Comment by jkowalleck** [comment]
The CycloneDX way of documenting patches would be [Pedigree](https://cyclonedx.org/use-cases/#pedigree)'s `patches`

**Reply by prabhu** [reply]
Often the hotfixes are based on edition and version numbers. So best to link these documents using external references rather than attempting to mimic the Windows update logic. Once the fixes are applied, the generated obom would include all patches, so it is possible to document the end state.
